google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -335,6 +335,77 @@ def labels= labels
           patch_gapi! :labels
         end
 
+        ##
+        # The {EncryptionConfiguration} object that represents the default
+        # encryption method for all tables and models in the dataset. Once this
+        # property is set, all newly-created partitioned tables and models in
+        # the dataset will have their encryption set to this value, unless table
+        # creation request (or query) overrides it.
+        #
+        # Present only if this dataset is using custom default encryption.
+        #
+        # @see https://cloud.google.com/bigquery/docs/customer-managed-encryption
+        #   Protecting Data with Cloud KMS Keys
+        #
+        # @return [EncryptionConfiguration, nil] The default encryption
+        #   configuration.
+        #
+        #   @!group Attributes
```

Comment:
Is this extraneous?

Reply:
I don't think so. Dataset has other methods that are grouped in "Attributes"